<!DOCTYPE html>
<html lang="en">
<head>
<meta http-equiv="X-UA-Compatible" content="IE=Edge" />
<meta charset="utf-8" />
</head>

<body style="margin: 0;">

<div id="p1" style="overflow: hidden; position: relative; background-color: white; width: 3640px; height: 2572px;">

<!-- Begin shared CSS values -->
<style class="shared-css" type="text/css" >
.t {
	transform-origin: bottom left;
	z-index: 2;
	position: absolute;
	white-space: pre;
	overflow: visible;
	line-height: 1.5;
}
.text-container {
	white-space: pre;
}
@supports (-webkit-touch-callout: none) {
	.text-container {
		white-space: normal;
	}
}
</style>
<!-- End shared CSS values -->


<!-- Begin inline CSS -->
<style type="text/css" >

#t1_1{left:234px;bottom:2354px;letter-spacing:-0.03px;word-spacing:-0.2px;}
#t2_1{left:149px;bottom:2283px;letter-spacing:-0.54px;word-spacing:0.46px;}
#t3_1{left:441px;bottom:2307px;letter-spacing:-0.19px;}
#t4_1{left:479px;bottom:2283px;letter-spacing:-0.07px;word-spacing:-1.26px;}
#t5_1{left:824px;bottom:2307px;}
#t6_1{left:839px;bottom:2283px;letter-spacing:-0.07px;}
#t7_1{left:1165px;bottom:2307px;}
#t8_1{left:1180px;bottom:2283px;letter-spacing:-0.25px;word-spacing:-0.23px;}
#t9_1{left:1507px;bottom:2307px;}
#ta_1{left:1522px;bottom:2283px;letter-spacing:-0.07px;}
#tb_1{left:1707px;bottom:2307px;}
#tc_1{left:1722px;bottom:2283px;letter-spacing:-0.07px;}
#td_1{left:2164px;bottom:2307px;}
#te_1{left:2179px;bottom:2283px;letter-spacing:-0.07px;}
#tf_1{left:2391px;bottom:2307px;}
#tg_1{left:2407px;bottom:2283px;letter-spacing:-0.07px;}
#th_1{left:2726px;bottom:2307px;letter-spacing:-0.19px;}
#ti_1{left:212px;bottom:2210px;letter-spacing:0.17px;word-spacing:2.79px;}
#tj_1{left:212px;bottom:2184px;letter-spacing:0.17px;word-spacing:2.05px;}
#tk_1{left:1248px;bottom:2184px;letter-spacing:0.18px;word-spacing:2.06px;}
#tl_1{left:212px;bottom:2158px;letter-spacing:0.17px;}
#tm_1{left:167px;bottom:2056px;letter-spacing:-0.25px;}
#tn_1{left:420px;bottom:2056px;letter-spacing:-0.24px;word-spacing:0.04px;}
#to_1{left:167px;bottom:2019px;letter-spacing:-0.2px;}
#tp_1{left:1104px;bottom:2019px;letter-spacing:-0.21px;}
#tq_1{left:167px;bottom:1982px;letter-spacing:-0.22px;}
#tr_1{left:893px;bottom:1982px;letter-spacing:-0.2px;}
#ts_1{left:167px;bottom:1946px;letter-spacing:-0.19px;word-spacing:-0.09px;}
#tt_1{left:167px;bottom:1909px;letter-spacing:-0.21px;}
#tu_1{left:167px;bottom:1872px;letter-spacing:-0.21px;}
#tv_1{left:1645px;bottom:1872px;letter-spacing:-0.21px;}
#tw_1{left:167px;bottom:1836px;letter-spacing:-0.19px;}
#tx_1{left:167px;bottom:1799px;letter-spacing:-0.22px;word-spacing:0.03px;}
#ty_1{left:1880px;bottom:2074px;letter-spacing:-0.27px;}
#tz_1{left:2141px;bottom:2074px;letter-spacing:-0.2px;}
#t10_1{left:1880px;bottom:2037px;letter-spacing:-0.24px;word-spacing:-0.11px;}
#t11_1{left:2212px;bottom:2037px;letter-spacing:-0.26px;}
#t12_1{left:2559px;bottom:2037px;letter-spacing:-0.21px;}
#t13_1{left:1880px;bottom:2001px;}
#t14_1{left:1890px;bottom:2001px;letter-spacing:-0.31px;}
#t15_1{left:2631px;bottom:2001px;letter-spacing:-0.2px;}
#t16_1{left:1880px;bottom:1964px;letter-spacing:-0.24px;word-spacing:-0.11px;}
#t17_1{left:3343px;bottom:1964px;letter-spacing:-0.22px;}
#t18_1{left:1880px;bottom:1927px;letter-spacing:-0.2px;}
#t19_1{left:1880px;bottom:1891px;letter-spacing:-0.2px;word-spacing:0.01px;}
#t1a_1{left:1880px;bottom:1854px;letter-spacing:-0.2px;}
#t1b_1{left:1880px;bottom:1817px;letter-spacing:-0.2px;}
#t1c_1{left:1037px;bottom:396px;letter-spacing:-0.26px;}
#t1d_1{left:1261px;bottom:396px;letter-spacing:-0.19px;}
#t1e_1{left:1870px;bottom:396px;letter-spacing:-0.21px;}
#t1f_1{left:1037px;bottom:360px;letter-spacing:-0.24px;word-spacing:0.03px;}
#t1g_1{left:2140px;bottom:360px;letter-spacing:-0.19px;}
#t1h_1{left:1037px;bottom:323px;letter-spacing:-0.19px;}
#t1i_1{left:1037px;bottom:286px;letter-spacing:-0.21px;word-spacing:0.01px;}
#t1j_1{left:1037px;bottom:250px;letter-spacing:-0.21px;}
#t1k_1{left:1424px;bottom:250px;letter-spacing:-0.21px;}
#t1l_1{left:1965px;bottom:250px;}
#t1m_1{left:1037px;bottom:189px;letter-spacing:0.16px;word-spacing:0.01px;}
#t1n_1{left:1037px;bottom:164px;letter-spacing:0.17px;word-spacing:0.01px;}
#t1o_1{left:1249px;bottom:1586px;letter-spacing:-0.67px;}
#t1p_1{left:1459px;bottom:1586px;letter-spacing:-0.07px;word-spacing:-0.1px;}
#t1q_1{left:1249px;bottom:1531px;letter-spacing:-0.07px;}
#t1r_1{left:1476px;bottom:1531px;letter-spacing:-0.07px;}
#t1s_1{left:1249px;bottom:1476px;letter-spacing:-0.09px;}
#t1t_1{left:1575px;bottom:1476px;letter-spacing:-0.07px;}
#t1u_1{left:1249px;bottom:1421px;letter-spacing:-0.07px;word-spacing:-1.01px;}
#t1v_1{left:1249px;bottom:1366px;letter-spacing:-0.08px;}
#t1w_1{left:1249px;bottom:1256px;letter-spacing:-0.08px;}
#t1x_1{left:1677px;bottom:1256px;letter-spacing:-0.08px;}
#t1y_1{left:2138px;bottom:1256px;letter-spacing:-0.07px;}
#t1z_1{left:1249px;bottom:1201px;letter-spacing:-0.07px;}
#t20_1{left:1249px;bottom:1146px;letter-spacing:-0.43px;word-spacing:0.25px;}
#t21_1{left:1249px;bottom:1091px;letter-spacing:-0.07px;}
#t22_1{left:1249px;bottom:1036px;letter-spacing:-0.07px;word-spacing:-0.36px;}
#t23_1{left:1249px;bottom:981px;letter-spacing:-0.07px;}
#t24_1{left:1249px;bottom:926px;letter-spacing:-0.07px;}
#t25_1{left:1249px;bottom:816px;letter-spacing:-0.08px;}
#t26_1{left:1802px;bottom:816px;letter-spacing:-0.08px;}
#t27_1{left:1249px;bottom:761px;letter-spacing:-0.1px;}
#t28_1{left:1407px;bottom:761px;letter-spacing:-0.07px;}
#t29_1{left:1249px;bottom:706px;letter-spacing:-0.07px;}
#t2a_1{left:1249px;bottom:651px;letter-spacing:-0.07px;}
#t2b_1{left:1613px;bottom:651px;letter-spacing:-0.05px;}
#t2c_1{left:1707px;bottom:651px;letter-spacing:-0.08px;}
#t2d_1{left:1249px;bottom:596px;letter-spacing:-0.09px;}
#t2e_1{left:1470px;bottom:596px;}
#t2f_1{left:3024px;bottom:2287px;letter-spacing:-0.13px;}
#t2g_1{left:149px;bottom:513px;letter-spacing:0.17px;word-spacing:-0.01px;}
#t2h_1{left:2683px;bottom:519px;letter-spacing:0.2px;word-spacing:-0.05px;}
#t2i_1{left:2683px;bottom:490px;letter-spacing:0.16px;word-spacing:-0.02px;}
#t2j_1{left:2683px;bottom:460px;letter-spacing:0.2px;}

.s1{font-size:61px;font-family:TimesNewRomanPS-BoldMT_w;color:#000;}
.s2{font-size:46px;font-family:TimesNewRomanPS-BoldMT_w;color:#000;}
.s3{font-size:31px;font-family:TimesNewRomanPS-BoldMT_w;color:#000;}
.s4{font-size:21px;font-family:ArialMT_13;color:#000;}
.s5{font-size:31px;font-family:Arial-BoldMT_17;color:#000;}
.s6{font-size:31px;font-family:ArialMT_13;color:#000;}
.s7{font-size:31px;font-family:ArialMT_13;color:#0097A7;}
.s8{font-size:46px;font-family:Arial-BoldMT_17;color:#000;}
.s9{font-size:46px;font-family:ArialMT_13;color:#000;}
.sa{font-size:46px;font-family:ArialMT_13;color:#0097A7;}
.sb{font-size:40px;font-family:ArialMT_13;color:#000;}
.sc{font-size:24px;font-family:ArialMT_13;color:#000;}
</style>
<!-- End inline CSS -->

<!-- Begin embedded font definitions -->
<style id="fonts1" type="text/css" >

@font-face {
	font-family: Arial-BoldMT_17;
	src: url("fonts/Arial-BoldMT_17.woff") format("woff");
}

@font-face {
	font-family: ArialMT_13;
	src: url("fonts/ArialMT_13.woff") format("woff");
}

@font-face {
	font-family: TimesNewRomanPS-BoldMT_w;
	src: url("fonts/TimesNewRomanPS-BoldMT_w.woff") format("woff");
}

</style>
<!-- End embedded font definitions -->

<!-- Begin page background -->
<div id="pg1Overlay" style="width:100%; height:100%; position:absolute; z-index:1; background-color:rgba(0,0,0,0); -webkit-user-select: none;"></div>
<div id="pg1" style="-webkit-user-select: none;"><object width="3640" height="2572" data="1/1.svg" type="image/svg+xml" id="pdf1" style="width:3640px; height:2572px; -moz-transform:scale(1); z-index: 0;"></object></div>
<!-- End page background -->


<!-- Begin text definitions (Positioned/styled in CSS) -->
<div class="text-container"><span id="t1_1" class="t s1">BPO: The Bioassay Protocol Ontology, a semantic approach to elevate the reusability of AMR-related drug discovery data </span>
<span id="t2_1" class="t s2">Yojana Gadiya </span>
<span id="t3_1" class="t s3">1,2 </span>
<span id="t4_1" class="t s2">, Rakel Arrazuria </span>
<span id="t5_1" class="t s3">3 </span>
<span id="t6_1" class="t s2">, Jon Ulf Hansen </span>
<span id="t7_1" class="t s3">4 </span>
<span id="t8_1" class="t s2">, Danielle Welter </span>
<span id="t9_1" class="t s3">5 </span>
<span id="ta_1" class="t s2">, Fuqi Xu </span>
<span id="tb_1" class="t s3">6 </span>
<span id="tc_1" class="t s2">, Philippe Rocca-Serra </span>
<span id="td_1" class="t s3">7 </span>
<span id="te_1" class="t s2">, Nick Juty </span>
<span id="tf_1" class="t s3">8 </span>
<span id="tg_1" class="t s2">, Philip Gribbon </span>
<span id="th_1" class="t s3">1,2 </span>
<span id="ti_1" class="t s4">(1) Fraunhofer Institute for Translational Medicine and Pharmacology (ITMP), Schnackenburgallee 114, 22525 Hamburg, and Theodor Stern Kai 7, 60590 Frankfurt, Germany (2) Fraunhofer Cluster of Excellence for Immune-Mediated Diseases (CIMD), Theodor Stern Kai 7, 60590 Frankfurt, Germany (3) Division of Microbiology, </span>
<span id="tj_1" class="t s4">Paul-Ehrlich-Institut, Langen, Germany (4) Dept. of Bacteria, Parasites and Fungi, Statens Serum Institut, </span><span id="tk_1" class="t s4">2300 Copenhagen, Denmark (5) Luxembourg Centre for Systems Biomedicine, ELIXIR Luxembourg, University of Luxembourg, L-4367 Belval, Luxembourg (6) European Molecular Biology Laboratory, European Bioinformatics </span>
<span id="tl_1" class="t s4">Institute, Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, UK (7) University of Oxford e-Research Centre, Department of Engineering Science, 7 Keble road, OX1 3QG, Oxford, United Kingdom.(8) Department of Computer Science, The University of Manchester, Oxford Road, Manchester, UK </span>
<span id="tm_1" class="t s5">INTRODUCTION: </span><span id="tn_1" class="t s6">To enhance the reproducibility and usability of bioassays in antimicrobial resistance (AMR) </span>
<span id="to_1" class="t s6">associated antibacterial drug discovery and development, there is an </span><span id="tp_1" class="t s5">increasing need for standardisation of </span>
<span id="tq_1" class="t s5">bioassay metadata into machine-readable formats</span><span id="tr_1" class="t s6">. Such a standardisation process requires mapping bioassay data </span>
<span id="ts_1" class="t s6">to standard ontologies and can be performed at the study result output and protocol levels. At the result output level, </span>
<span id="tt_1" class="t s5">general ontologies such as the BioAssay Ontology (BAO) already exist, but antibacterial drug discovery and </span>
<span id="tu_1" class="t s5">AMR-specific ontologies that can aid in the standardisation process at the protocol level are missing. </span><span id="tv_1" class="t s6">Hence, we </span>
<span id="tw_1" class="t s6">propose a new application-specific ontology that will enable researchers to convert unstructured bioassay protocol data </span>
<span id="tx_1" class="t s6">into structured machine-readable formats thereby promoting their overall reusability. </span>
<span id="ty_1" class="t s5">METHODOLOGY: </span><span id="tz_1" class="t s6">The Bioassay Protocol Ontology (BPO) was constructed in accordance with the ontology build life </span>
<span id="t10_1" class="t s6">cycle. The ROBOT tool ( </span><span id="t11_1" class="t s7">http://robot.obolibrary.org/</span><span id="t12_1" class="t s6">) and the Ontology Development Kit ODK </span>
<span id="t13_1" class="t s6">( </span><span id="t14_1" class="t s7">https://github.com/INCATools/ontology-development-kit</span><span id="t15_1" class="t s6">) were used by scientific domain experts to programmatically </span>
<span id="t16_1" class="t s6">develop the ontology. The BPO is accessible to the AMR community in the Ontology Web Language (OWL) </span><span id="t17_1" class="t s6">and Open </span>
<span id="t18_1" class="t s6">Biomedical Ontologies (OBO) formats. </span>
<span id="t19_1" class="t s6">We tested 10 bioassay specific ontologies within this project to check the extend of extraction of information from </span>
<span id="t1a_1" class="t s6">natural language processing (NLP) models. But none of them were able to provide a </span>
<span id="t1b_1" class="t s6">complete overview of underlying information. </span>
<span id="t1c_1" class="t s5">CONCLUSION: </span><span id="t1d_1" class="t s6">The development of BPO is a first attempt to </span><span id="t1e_1" class="t s5">standardise and organise information within in-vivo </span>
<span id="t1f_1" class="t s5">AMR-related drug discovery bioassay protocol data in a structured manner. </span><span id="t1g_1" class="t s6">BPO will allow researchers to </span>
<span id="t1h_1" class="t s6">capture information regarding experimental details such as the type of mouse model, the bacterial strain, and the sex </span>
<span id="t1i_1" class="t s6">and growth phases of mouse and bacteria respectively from the protocol. We have made the ontology available for </span>
<span id="t1j_1" class="t s6">community reuse on GitHub </span><span id="t1k_1" class="t s7">https://github.com/Fraunhofer-ITMP/bpo</span><span id="t1l_1" class="t s6">. </span>
<span id="t1m_1" class="t s4">This research was funded by the EU/EFPIA Innovative Medicines Initiative 2 Joint Undertaking project Collaboration for prevention and treatment of MDR bacterial </span>
<span id="t1n_1" class="t s4">infections (COMBINE) IMI2 JU (grant agreement # 853967) and FAIRplus (grant agreement #802750). </span>
<span id="t1o_1" class="t s8">RESULTS</span><span id="t1p_1" class="t s9">: The BPO was initially established in order to </span>
<span id="t1q_1" class="t s9">extend the </span><span id="t1r_1" class="t s8">interoperability and reusability of in-vivo </span>
<span id="t1s_1" class="t s8">AMR bioassay </span><span id="t1t_1" class="t s9">protocol data generated within the </span>
<span id="t1u_1" class="t s9">Innovative Medicines Initiative (IMI) AMR Accelerator </span>
<span id="t1v_1" class="t s9">project, COMBINE. </span>
<span id="t1w_1" class="t s9">Thus, we created an </span><span id="t1x_1" class="t s8">application ontology </span><span id="t1y_1" class="t s9">through the </span>
<span id="t1z_1" class="t s9">combination of existing terms from prominent ontologies </span>
<span id="t20_1" class="t s9">such as NCBI Taxonomy and PATO along with new </span>
<span id="t21_1" class="t s9">terms around the animal model-specific details, such as </span>
<span id="t22_1" class="t s9">mouse inbred and outbred status. Alongside addition of </span>
<span id="t23_1" class="t s9">new terms, several terms have additional been </span>
<span id="t24_1" class="t s9">requested to the parent reference ontologies. </span>
<span id="t25_1" class="t s9">Overall, BPO ontology has </span><span id="t26_1" class="t s8">209 classes and 2,242 </span>
<span id="t27_1" class="t s8">axioms</span><span id="t28_1" class="t s9">. In the coming months, we aim to integrate </span>
<span id="t29_1" class="t s9">BPO into our information retrieval environment (GRIT42 </span>
<span id="t2a_1" class="t sa">https://grit42.com/</span><span id="t2b_1" class="t s9">) for </span><span id="t2c_1" class="t s8">protocol-level annotation of </span>
<span id="t2d_1" class="t s8">bioassays</span><span id="t2e_1" class="t s9">. </span>
<span id="t2f_1" class="t sb">Abstract number : EA71 </span>
<span id="t2g_1" class="t sc">Figure 1: The overview of the major classes included in the BPO ontology. </span>
<span id="t2h_1" class="t sc">Figure 2: The additional classification and metadata annotation in BPO </span>
<span id="t2i_1" class="t sc">ontology. This includes the classification of bacteria strain and annotation of </span>
<span id="t2j_1" class="t sc">mouse strains based on their breeding or immunological classification </span></div>
<!-- End text definitions -->


</div>
</body>
</html>
